website/docs/API.md

```diff
@@ -782,7 +783,8 @@ it('should use context value', () => {
 ```ts
 type Config = {
   asyncUtilTimeout: number;
-  defaultDebugOptions: Partial<DebugOptions>
+  defaultDebugOptions: Partial<DebugOptions>;
+  hidden: boolean;
```

Comment:
Isn't the option called default hidden?
```suggestion
  defaultHidden: boolean;
```

Reply:
nope it's called hidden for now, I aligned with asyncUtilTimeout which did not have the prefix hidden 
but now we also have defaultDebugOptions...
I think we should have the same naming pattern for the three options

Reply:
Can you switch it to `defaultHidden`. Both names, `asyncUtilTImeout` and `defaultHidden`, are taken from DOM testing library: https://testing-library.com/docs/dom-testing-library/api-configuration/#defaulthidden

Reply:
alright!